### DISCUSSION TITLE
Explore controlling how .md/.mdx is formatted

### DISCUSSION THREAD

**Discussion by jamespohalloran** [tinacms/tinacms#2578] [General]
## Summary

When I save my MDX file, all of my horizontal-rule's have their formatting changed:

<img width="860" alt="Screen Shot 2022-02-11 at 10 16 50 AM" src="https://user-images.githubusercontent.com/3323181/153607606-9201cffb-9257-41c4-a0e6-ecb78ae107dd.png">

This would be the same for unordered lists, bold/italic,etc. 

This isn't a huge problem, but it can be jarring when you go to make your first commit and there's a huge diff.

At a minimum, it might be good to have a document that shows the flavor of markdown that gets exported with Tina.
It would also be great if the user could control this, (although I'm unclear on the degree to which this matters for devs).

Does this matter to people? 
If so, do we have thoughts on how this could be configured?

**Comment by mitchmac** [comment]
@jamespohalloran Folks who look at their content diffs will find the formatting changes annoying. Is it only a situation for content that is authored outside of the rich text editor?

**Reply by jamespohalloran** [reply]
I'm not really sure what the solution is at this point (or where it should be configured). But this would apply to both rich-text content, and "string" content edited with the markdown editor

**Comment by jeffsee55** [comment]
We can pull in [GFM](https://github.com/remarkjs/remark-gfm) formatting and offer that as an option (which I think is the difference between what you're seeing `---` vs `***`). But there's likely always going to be a difference between actual user input and what we generate, for example a lot of VSCode users have markdown's prettier formatter.

**Comment by LaughingZhu** [comment]
Just wondering if there is any update on this escape character issue. @jamespohalloran 
I implemented the callout feature in my blog, like
```
:::tip[help]
there is  some text
:::
```
after formatting changed

```
\:::tip\[help]
there is  some text
\:::
```

it's not worked
